packages/main/src/Table.ts

```diff
@@ -80,6 +82,7 @@ type TableRowClickEventDetail = {
  *
  * The `ui5-table` can be enhanced in its functionalities by applying different features.
  * Features can be slotted into the `features` slot, to enable them in the component.
+ * Features need to be imported separately, as they are not enabled by default.
```

Comment:
It would be nice to give a sample here 
e.g. in order to use the selection functionality, you need to import the following module:
import "@ui5/webcomponents/blabla"